docs/UserGuide.md

```diff
@@ -164,6 +164,30 @@ Exits TrackAScholar.
 
 Format: `exit`
 
+### Removing applicants by status : `remove`
+
+All applicants with the corresponding statuses stored in TrackAScholar are removed.
+
+Format: `remove [ACCEPTED/REJECTED]`
+
+* The user selects the applicant(s) with application status of either `ACCEPTED` or `REJECTED` to be removed from TrackAScholar. 
+
+Examples:
+*  `remove accepted` removes all entries with ACCEPTED as an application status from TrackAScholar.
+*  `remove rejected` removes all entries with REJECTED as an application status from TrackAScholar.
+
+### Removing applicants by status : `sort`
```

Comment:
Header name doesn't correspond with command name.

Reply:
Alright i will resolve it and let you know when i am done!